salt/metalk8s/addons/monitoring/kube-state-metrics/upstream.sls

```diff
@@ -219,11 +219,11 @@ spec:
         name: kube-state-metrics
         resources:
           limits:
-            cpu: 102m
```

Comment:
This part should be reverted in a dedicated commit, for "historical" purpose (you can probably use `git revert`).